packages/pwa-buildpack/envVarDefinitions.json

```diff
@@ -0,0 +1,256 @@
+{
+    "sections": [
+        {
+            "name": "Connecting to a Magento store",
+            "variables": [
+                {
+                    "name": "MAGENTO_BACKEND_URL",
+                    "type": "url",
+                    "desc": "Connect to an instance of Magento 2.3 by specifying its public domain name.",
+                    "example": "https://magento.localhost/"
+                }
+            ]
+        },
+        {
+            "name": "Custom local origin",
+            "variables": [
+                {
+                    "name": "CUSTOM_ORIGIN_ENABLED",
+                    "type": "bool",
+                    "desc": "Get or create a secure, unique hostname/port combination and a trusted SSL certificate for local development.",
+                    "default": true
+                },
+                {
+                    "name": "CUSTOM_ORIGIN_ADD_UNIQUE_HASH",
+                    "type": "bool",
+                    "desc": "Add a unique hash string to the custom origin, based on filesystem location. This naturally separates domains when running multiple project folders on one developer machine.",
+                    "default": true
+                },
+                {
+                    "name": "CUSTOM_ORIGIN_SUBDOMAIN",
+                    "type": "str",
```

Comment:
Can we use "string" instead of "str" in this file? It reads easier.

Reply:
From another comment:
>It would, but that field is used as an [`envalid` validator type](https://www.npmjs.com/package/envalid#validator-types), so we're stuck with their name scheme.